src/main/resources/resources/logisim/strings/std/std.properties

```diff
@@ -763,7 +763,8 @@ TTL74158 = 74158: quad 2-line to 1 line data selector, inverted output
 TTL74139 = 74139: Dual 2-line to 4-line decoder
 TTL7413 = 7413: dual 4-input NAND gate(schmitt-trigger)
 TTL7414 = 7414: hex inverter (schmitt-trigger)
-TTL74161 = 74161: 4-bit sync counter with clear
+TTL74161 = 74161: 4-bit sync counter with async clear
```

Comment:
Pleaase `asynchronous` instead of `async` as it looks too similar to "sync" on the list.

Reply:
I agree, however, the title is already way too long, in the library with standard logisim setting you not even see the ```with ....``` part. We should maybe have other descriptions here, but I'm not quite clear what would make sense. Leaving it as is for the moment